tests/shop/test_01.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+
+from src.shop.__init__ import *
+
+
+class Place_order(unittest.TestCase):
+    def test_01(self):
+        bob = Customer("Bob")
+        order = place_order(bob, [tv, 1], [iPhone, 1], [chocolate, 2])
+        self.assertEqual(1510, order.value)
+
+    def test_02(self):
```

Comment:
Metody testowe mogą także otrzymać pełne nazwy, które jasno zobrazują co jest testowane. Spójrz na inne moduły testowe.

Reply:
To samo tyczy się modułów testowych. Dobrze jest je nazywać w taki sposób, żeby łatwo było je znaleźć w przyszłości np. `test_shop.py` albo bardziej specyficznie `test_place_order.py`.